pyproject.toml

```diff
@@ -113,8 +112,15 @@ select = [
 ignore = [
   "ANN101", # no type for self
   "ANN102", # no type for cls
-  "SIM108", # allow if-else in place of ternary
+  "E111", # indentation is not a multiple of four
+  "E117", # over-indented
+  "Q000", # bad quotes inline string
+  "Q001", # bad quotes multiline string
+  "Q002", # bad quotes docstring
+  "Q003", # avoidable escaped quotes
   "RET505", # allow else after return
+  "SIM108", # allow if-else in place of ternary
+  "W191", # indentation contains tabs
```

Comment:
indentation should never contain tabs

Reply:
oh cool, maybe group those together and have a comment around them pointing to that doc?